src/components/ui/autocomplete/autocomplete.component.tsx

```diff
@@ -190,12 +190,30 @@ export class Autocomplete extends React.Component<AutocompleteProps, State> {
     return React.cloneElement(info.item, { onPress: () => this.onItemPress(info.index) });
   };
 
+  private renderAnchorInputElement = (props: InputProps): InputElement => {
+    return (
+      <View>
+        <Input
+          {...props}
+          ref={this.inputRefAnchor}
+          testID='@autocomplete/input-anchor'
+          showSoftInputOnFocus={false}
+          onFocus={this.onAnchorInputFocus}
+          onSubmitEditing={this.onInputSubmitEditing}
+        />
+      </View>
+    );
+  };
+
   private renderInputElement = (props: InputProps): InputElement => {
     return (
       <View>
         <Input
           {...props}
           ref={this.inputRef}
+          testID='@autocomplete/input'
```

Comment:
Shouldn't we allow users to define this `testID` themselves? I'm concerned they won't be able to run tests correctly with such `testID` definition. Inputs are usually essential component for test cases.